doc/guide/Guide_Installation.ipynb

```diff
@@ -73,28 +77,33 @@
     "\n",
     "\n",
     "#### Workaround option B:\n",
-    "In a terminal window, enter:\n",
-    "\n",
+    "Start Anaconda, create a new python_env environment (just click create and enter the name, then press Create) and install latest Spyder in there (currently 4.2.5). Start this version of Spyder. After Spyder has started, navigate to *Preferences > Python Interpreter > Use the following interpreter* and paste the output of the following terminal command into the text box:\n",
     "```bash\n",
     "conda activate climada_env \n",
-    "conda install spyder-kernels \n",
     "python -c \"import sys; print(sys.executable)\" \n",
-    "--> returns a path, like /Users/XXX/opt/anaconda3/envs/climada_env/bin/python\n",
+    "# this returns a path, like /Users/XXX/opt/anaconda3/envs/climada_env/bin/python\n",
     "```\n",
+    "Now, obtain the version of the package `spyder-kernels` (currently 1.10.2) that has automatically been installed along with Spyder in the Anaconda environment and install the same version of `spyder-kernels` in the `climada_env` environment:\n",
     "\n",
-    "Start Anaconda, create a new python_env environment (just click create and enter the name, then press Create) and install latest Spyder in there (currently 4.1.5), start this Spyder, then, after Spyder has started, navigate to Preferences > Python Interpreter > Use the following interpreter and paste the path from above (e.g. /Users/XXX/opt/anaconda3/envs/CLIMADA_env/bin/python) into the text box. Start a new IPython console and run ``tests_install.py``.\n",
+    "```bash\n",
+    "conda activate climada_env \n",
+    "conda install spyder-kernels=1.10.2\n",
+    "```\n",
+    "\n",
+    "Now, start a new IPython console in Spyder and run ``tests_install.py``.\n",
     "\n",
     "## FAQs\n",
     "\n",
-    "* ModuleNotFoundError; CLIMADA libraries are not found. Try to include *climada_python-x.y.z* path in the environment *climada_env* path as suggested in Section 2 of [Install environment with Anaconda]. If it does not work you can always include the path manually before executing your code:\n",
+    "* `ModuleNotFoundError`; the CLIMADA packages are not found. Try to include the paths to `climada_python` and `climada_petals` into the environment `climada_env` using `pip install -e` as suggested in [Install environment with Anaconda or Miniconda]. If it does not work, you can always include the path manually before executing your code:\n",
```

Comment:
Before this PR, this section used to refer to a now non-existent section of the installation instructions.